src/timedate/timedatectl.c

```diff
@@ -261,6 +261,15 @@ static int set_local_rtc(int argc, char **argv, void *userdata) {
         if (b < 0)
                 return log_error_errno(b, "Failed to parse local RTC setting '%s': %m", argv[1]);
 
+        if (b == 1)
+		printf("\n%s"
+                       "Warning: The system is configured to read the RTC time in the local time zone.\n"
+                       "         This mode cannot be fully supported. It will create various problems\n"
+                       "         with time zone changes and daylight saving time adjustments. The RTC\n"
+                       "         time is never updated, it relies on external facilities to maintain it.\n"
+                       "         If at all possible, use RTC in UTC by calling\n"
+                       "         'timedatectl set-local-rtc 0'.%s\n", ansi_highlight(), ansi_normal());
```

Comment:
It doesn't make sense to copy the warning verbatim or to suggest the command, when the user is already calling the command, but with a different value.

Maybe something like this:
"Warning: The system is being configured to read the RTC time in the local time zone.
                   ...
                   If at all possible, use RTC in UTC instead."